Source/WebCore/contentextensions/ContentExtensionParser.cpp

```diff
@@ -313,6 +325,9 @@ static Expected<Vector<ContentExtensionRule>, std::error_code> loadEncodedRules(
 
     Vector<ContentExtensionRule> ruleList;
 
+#if ENABLE(DNR_ON_RULE_MATCHED_DEBUG)
+    size_t rulesWithIdentifiers = 0;
```

Comment:
Nit: From the variable name, it seems like this would be a set of rules. `numberOfRulesWithIdentifiers`/`rulesWithIdentifiersCount` might make this a bit more clear.